src/util/address.rs

```diff
@@ -329,7 +499,7 @@ impl Address {
     }
 
     /// Check whether or not the address is following Bitcoin
-    /// standardness rules.
+    /// standard rules.
```

Comment:
Not a native speaker but the original seemed correct. The rules check if the address is standard.

Reply:
Agreed.